nemo/collections/tts/torch/tts_dataset.yaml

```diff
@@ -22,7 +22,7 @@ tts_dataset:
     _target_: nemo_text_processing.text_normalization.normalize.Normalizer
     lang: en
     input_case: cased
-    whitelist: "nemo_text_processing/text_normalization/en/data/whitelist/lj_speech.tsv"
+    whitelist: "scripts/tts_dataset_files/en/whitelist_lj_speech.tsv"
```

Comment:
ls: scripts/tts_dataset_files/en/whitelist_lj_speech.tsv: No such file or directory

Reply:
discussed offline. removed this argument.